lib/dispatcher.rb

```diff
@@ -29,15 +29,15 @@ def email_sent_confirmation(proposal)
 
   def deliver_new_proposal_emails(proposal)
     proposal.currently_awaiting_approvals.each do |approval|
-      self.email_approver(approval)
+      email_approver(approval)
     end
-    self.email_observers(proposal)
-    self.email_sent_confirmation(proposal)
+
+    email_observers(proposal)
+    email_sent_confirmation(proposal)
   end
 
   def deliver_attachment_emails(proposal)
     proposal.users.each do |user|
-      # do not send email to approvers who have not yet heard about the proposal
```

Comment:
Yep, totally understand the value of the test, but the problem the comment solves for is "I'm looking at this code, but don't understand what it's doing," which tests don't really help with. Ideal would be making a self-descriptive method to retrieve the recipients, which is what the story linked to above is about.